wiki/Help_Centre/Gameplay/en.md

```diff
@@ -0,0 +1,283 @@
+
+**Some background processes can fight with osu! for resources and make the game very unpleasant to play.**
+
+Try closing some programs that you don't need open while playing the game. You can check which programs are using up the most resources on your computer by pressing `Ctrl` + `Shift` + `Esc` to open the Task Manager.
+
+You can also try checking to see how much priority your computer is giving to osu!. To check or change this, do the following:
+
+*Warning: We do not reccomend selecting priority `Realtime` as this may cause resource issues within your system. Your computer may even block you from selecting this setting.*
+
+- Open the Task Manager.
+- Right click on osu!.
+- Click `Go to details`.
+- Right click on the highlighted program (this should be called `osu!.exe`).
+- Click on `Set priority`.
+- If you want more resources to be sent to osu!, click on one of the six options shown in the sidebar. 
+
+### Is your computer secure?
+
+**Viruses, spyware, malware or other unwanted software can cause serious performance problems for even high-end computers.**
+
+Please be sure to keep your computer clean and safe, and you might notice an improvement.
+
+### How old is your computer?
+
+**Very old computers that use even older Intel Integrated graphics chipsets may not be powerful to handle osu! at all, even on the lowest settings.**
+
+The only way to deal with this one is to upgrade your computer or graphics card!
+
+### Are your drivers up-to-date?
+
+**Old drivers can cause serious performance issues. Make sure you keep all of your drivers up-to-date for everything, wherever possible.**
+
+Graphics card drivers are particularly important for osu!, as well as sound card drivers.
+
+### Are you using a USB hub to connect lots of USB devices to your computer through one cable?
+
+**Connecting too many devices to one port via a splitter can cause input lag, which is hard to detect, but can seriously throw off your game.**
+
+An easy solution to this is to try unplugging your USB hub briefly. If your lag disappears, you'll know why!
+
+### Are you running osu! in Windowed or Borderless mode?
+
+**Both Windowed and Borderless mode introduce a *considerable* amount of input delay.** 
+
+We reccomend using Fullscreen as it is optimal for the smoothest gameplay experience. <!-- this line could be bolded with the line above, or stay down here, I can't decide tho (remove comment before merging) -->
+
+If you are using windowed or borderless mode to sync up your monitor's refresh rate, we reccomend you try selecting the `VSync` option under `Frame Limiter` first! <!-- this line and the line below are only one sentence long but they don't really fit together on the same line (remove comment before merging) -->
+
+If you are using windowed mode to lower the "sensitivity" of your tablet, we reccomend that you try adjusting your tablet area (which you can learn to do in the "Performance" section of this article!).
```

Comment:
```suggestion
If you are using windowed mode to lower the "sensitivity" of your tablet, we recommend that you try adjusting your tablet area (which you can learn to do in the "Performance" section of this article!).
```
Also, the performance section is no longer on the same page as this one, so it should probably be linked to directly instead of just mentioned. Not sure if we want to change the wording of the bit in parenthesis at the same time, so leaving that out of the suggested change.